packages/shared/src/components/history/ReadingHistoryList.tsx

```diff
@@ -87,7 +81,7 @@ export default function ReadHistoryList({
       <InfiniteScrollScreenOffset ref={infiniteScrollRef} />
       <PostOptionsReadingHistoryMenu
         post={readingHistoryContextItem?.post}
-        onShare={() => openSharePost(readingHistoryContextItem.post)}
```

Comment:
Had to fix a lot of types from the hooks related to this as the object is being defined as `any`.